### DISCUSSION TITLE
Create account does not work no longer.

### DISCUSSION THREAD

**Discussion by burakkarkc** [supabase/supabase#31331] [Questions] [answered]
It was working but, does not work now? Why? I did not even touch the user table, or user sign up screen. The button creates user in auth section but not in the table. I did everyting right and I am very sure on it. And also I checked everything again, it seems very true.

**Comment by GaryAustin1** [comment]
You need to provide more info on what you are referring to.  Which user table (auth or a public one you have)?  What sign up screen?  Which button?  What is the auth section versus not in table?

Do you mean the dashboard authentication tab shows the user but the auth.users table does not?  Don't think that is possible.

Also you should check the logs for auth errors and Postgres errors depending on what you are doing.

This is working for me if that is what you mean by create user...
![image](https://github.com/user-attachments/assets/04b64477-5f1a-49a5-a911-74b7d880de3b)

**Comment by burakkarkc** [comment]
![image](https://github.com/user-attachments/assets/bb62c8a2-e552-4060-a1ea-c00b11ba86aa)
![image](https://github.com/user-attachments/assets/8495bb87-5139-4f49-befb-6bf4fe180c84)
![image](https://github.com/user-attachments/assets/92dbc0de-5602-456c-bb97-12e22c8d4699)

**Comment by burakkarkc** [comment, marked as answer]
Okay, Got the solution. You have to "Get Schema" to keep current the datas if you come across this situation, then you have to update somethings on app it related to yours. It does not important what app version you use, it effects all of the versions unfortunately, so "Revert" will not be useful for it.  It happens sometimes if you do big changes on app or database... Fun but fact. Good night you all.
